exporter/stackdriverexporter/config_test.go

```diff
@@ -52,5 +52,6 @@ func TestLoadConfig(t *testing.T) {
 			Endpoint:                   "test-endpoint",
 			NumOfWorkers:               3,
 			SkipCreateMetricDescriptor: true,
+			Insecure:                   true,
```

Comment:
Name this UseInsecure like the GrpcSettings? So in case we change to use that it is not a breaking change?

Reply:
Done.